common/src/main/java/com/wynntils/mc/event/PlayerFeatureRenderTranslucentCheckEvent.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.event;
+
+import com.wynntils.mc.extension.PlayerModelExtension;
+import net.minecraft.world.entity.LivingEntity;
+
+public class PlayerFeatureRenderTranslucentCheckEvent extends LivingEntityRenderTranslucentCheckEvent {
+    private final PlayerModelExtension playerModelExtension;
+
+    protected PlayerFeatureRenderTranslucentCheckEvent(
+            boolean translucent, LivingEntity entity, float translucence, PlayerModelExtension playerModelExtension) {
+        super(translucent, entity, translucence);
+        this.playerModelExtension = playerModelExtension;
+    }
+
+    public PlayerModelExtension getPlayerModelExtension() {
+        return this.playerModelExtension;
+    }
+
+    /**
+     * Fired when {@link net.minecraft.client.renderer.entity.layers.CapeLayer} checks whether a living
+     * entity cape should be rendered translucent or not
+     */
+    public static class Cape extends PlayerFeatureRenderTranslucentCheckEvent {
+        public Cape(
+                boolean translucent,
+                LivingEntity entity,
+                float translucence,
+                PlayerModelExtension playerModelExtension) {
+            super(translucent, entity, translucence, playerModelExtension);
+        }
+
+        /**
+         * Translucence value needs to pass into {@link net.minecraft.client.model.PlayerModel} class,
+         * because {@link net.minecraft.client.model.PlayerModel#renderCloak(PoseStack, VertexConsumer, int, int)} method does not accept {@code color}
+         * argument
+         * @param translucence
+         */
+        @Override
+        public void setTranslucence(float translucence) {
+            super.setTranslucence(translucence);
```

Comment:
So when we set the cape translucence, we also set the player/entity translucence? Is that expected/wanted behavior?

Reply:
Actually, I don't think that is what is happening. It is just propagating the value up the strange event inheritance hierarchy.

But the code is way too convoluted for what it is supposed to be doing, so I understand that you interpret it as this. (And then again, it might be me who is wrong about this as well...)